tests/Unit/Utilities/Test_PrettyType.cpp

```diff
@@ -249,190 +272,204 @@ SPECTRE_TEST_CASE("Unit.Utilities.PrettyType.short_name", "[Utilities][Unit]") {
-  CHECK(pretty_type::short_name<Template2<std::nullptr_t, int>>() ==
+  CHECK(NameFunc::template name<Template2<std::nullptr_t, int>>() ==
         "Template2");
-  CHECK(pretty_type::short_name<Template2<std::nullptr_t, Global>>() ==
+  CHECK(NameFunc::template name<Template2<std::nullptr_t, Global>>() ==
         "Template2");
-  CHECK(pretty_type::short_name<Template2<std::nullptr_t, TestType>>() ==
+  CHECK(NameFunc::template name<Template2<std::nullptr_t, TestType>>() ==
         "Template2");
-  CHECK(pretty_type::short_name<Template2<std::nullptr_t, Special>>() ==
+  CHECK(NameFunc::template name<Template2<std::nullptr_t, Special>>() ==
         "Template2");
-  CHECK(pretty_type::short_name<NullptrTemplate<nullptr>>() ==
+  CHECK(NameFunc::template name<NullptrTemplate<nullptr>>() ==
         "NullptrTemplate");
 
   // Enum non-type template parameters
-  CHECK(pretty_type::short_name<
+  CHECK(NameFunc::template name<
             GlobalEnumTemplate<Test_PrettyType_Enum::Zero>>() ==
         "GlobalEnumTemplate");
-  CHECK(pretty_type::short_name<
+  CHECK(NameFunc::template name<
             GlobalEnumTemplate<Test_PrettyType_Enum::One>>() ==
         "GlobalEnumTemplate");
-  CHECK(pretty_type::short_name<
+  CHECK(NameFunc::template name<
             Template2<GlobalEnumTemplate<Test_PrettyType_Enum::One>,
                       GlobalEnumTemplate<Test_PrettyType_Enum::Two>>>() ==
         "Template2");
-  CHECK(pretty_type::short_name<LocalEnumTemplate<LocalEnum::Zero>>() ==
+  CHECK(NameFunc::template name<LocalEnumTemplate<LocalEnum::Zero>>() ==
         "LocalEnumTemplate");
-  CHECK(pretty_type::short_name<LocalEnumTemplate<LocalEnum::One>>() ==
+  CHECK(NameFunc::template name<LocalEnumTemplate<LocalEnum::One>>() ==
         "LocalEnumTemplate");
   CHECK(
-      pretty_type::short_name<Template2<LocalEnumTemplate<LocalEnum::One>,
+      NameFunc::template name<Template2<LocalEnumTemplate<LocalEnum::One>,
                                         LocalEnumTemplate<LocalEnum::Two>>>() ==
       "Template2");
-  CHECK(pretty_type::short_name<LocalEnumTemplate<LocalEnum::Negative>>() ==
+  CHECK(NameFunc::template name<LocalEnumTemplate<LocalEnum::Negative>>() ==
         "LocalEnumTemplate");
 
   // Complicated Test
-  CHECK(pretty_type::short_name<OuterTemplate<
+  CHECK(NameFunc::template name<OuterTemplate<
             Template2<OuterTemplate<Global>::InnerTemplate<TestType>,
                       OuterTemplate<Global>::InnerTemplate<TestType>>>::
                                     InnerTemplate<NonTypeType<1, Global>>>() ==
         "InnerTemplate");
 
   // Long test
-  CHECK(pretty_type::short_name<tmpl::range<int, 0, 1000>>() == "list");
+  CHECK(NameFunc::template name<tmpl::range<int, 0, 1000>>() == "list");
+}
+
+SPECTRE_TEST_CASE("Unit.Utilities.PrettyType.short_name", "[Utilities][Unit]") {
+  test_name_func<ShortName>();
 }
+
+SPECTRE_TEST_CASE("Unit.Utilities.PrettyType.name", "[Utilities][Unit]") {
```

Comment:
Please combine the two test cases into one. Test cases have a nonnegligible startup time.